### ISSUE TITLE
Change message

### ISSUE THREAD

**Issue by TimHarrio** [gvlfm78/BukkitOldCombatMechanics#179]
Can you add the ability to change the message of /ocm toggle,
and add the commande /ocm 1.8 and /ocm 1.9 to select the type of pvp.

**Comment by Rayzr522** [comment]
Ah, so you mean a way to simply turn OCM on and off rather than toggle?

And yes, I can certainly add translatable messages.

**Comment by gvlfm78** [comment]
So like a global toggle?

**Comment by TimHarrio** [comment]
No, at the place of do /ocm toggle for change the mode, we can select directly the mode with "/ocm 1.8" or "/ocm 1.9"

**Comment by Rayzr522** [comment]
@gvlfm78 is it possible that per-user combat could be implemented by wrapping all event listeners in a check that would cancel the listener if the player still wanted to use 1.9 combat? Not sure what you think of that approach, it might be a bit too invasive as there may still be some event listeners we would want to function whether the player is using 1.9 or 1.8 combat.

**Comment by kernitus** [comment]
Implemented from commit 120bf2dc
